### PR TITLE
Merge tool lib 2.x to master

### PR DESCRIPTION
**Pull request by KonstantinTyukalov**
Changes:
- Bumped tool-lib to 2.0.0
- Added changelog document
- Increased default mocha timeout to 20sec
- Downgraded mocha to 6.2.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
   "dependencies": {
     "@types/semver": "^5.3.0",
     "@types/uuid": "^3.4.5",
-    "azure-pipelines-task-lib": "^3.1.10",
+    "azure-pipelines-task-lib": "^4.0.0-preview",
```
Comment (max-zaytsev) [inline package.json:28]: ^4.1.0

Comment (KonstantinTyukalov) [reply package.json]: Updated, thanks

**Review by kirill-ivlev — APPROVED** [review]

**Review by max-zaytsev — APPROVED** [review]

**Review by wojgasior — APPROVED** [review]

**Pull request merged**